<launch>

<!-- args --> 
    <!-- Behzad's workstation ip --> 
    <!-- arg name="host_ip" value="10.157.34.101" /--> 
    <arg name="host_ip" value="10.157.90.51" /> 
    <!-- Hasans's workstation ip --> 
    <!-- arg name="host_ip" value="10.157.34.208" /--> 
    
    <!-- setting DEBUG to false prevents publishing and subscribing to rviz /--> 
    <!-- also planning to prevent stdio stuff(if they really matter)--> 
    <arg name="DEBUG" value="true" /> 
    <param name="ros_DEBUG" type="bool" value="$(arg DEBUG)" />

    <!-- set this to "" if you don't want perf -->
    <arg name="my_launch_prefix" value="" /> 
    <!-- arg name="my_launch_prefix" value="/home/nvidia/install/perf/perf record -g fp"/ -->
    


    <!-- starting nodes --> 
    <!-- control drone --> 
    <node pkg="follow_the_leader" type="pid_node" name="pid_node" output="screen">
        <param name="vx__P__global" value=".0056"/>
        <param name="vy__P__global" value=".0056"/>
        <param name="vz__P__global" value="1"/>

        <param name="vx__I__global" value=".001"/>
        <param name="vy__I__global" value=".001"/>
        <param name="vz__I__global" value=".001"/>

        <param name="vx__D__global" value=".01"/>
        <param name="vy__D__global" value=".01"/>
        <param name="vz__D__global" value=".01"/>

        <param name="image_w__global" value="400"/>
        <param name="image_h__global" value="800"/>
        
        <param name="ip_addr__global" value="$(arg host_ip)"/>
    </node>

</launch>
